test/cmdlineargs.jl

```diff
@@ -97,6 +97,11 @@ let exename = `$(Base.julia_cmd()) --startup-file=no`
         @test startswith(read(`$exename --help`, String), header)
     end
 
+    # --project
+    let header = "    Status `~/foo/Project.toml`"
+        @test startswith(read(`$exename --project=~/foo -e'using Pkg; Pkg.status()'`, String), header)
```

Comment:
Best not to use Pkg in these tests, you can test the output of `Base.active_project()` directly. How about something like the following:
```julia
@test occursin(abspath(expanduser("~/foo")),
          readchomp(`$exename --project='~/foo' -E 'Base.active_project()'`))
```